connectors/google/install.js

```diff
@@ -24,35 +24,34 @@ google_scope=${ctx.body.configuration.google_scope || ''}
 # Allowed returnTo URLs. Comma delimited. Use wildcard (*) at the end of a URL to allow for a prefix match
 fusebit_allowed_return_to=${ctx.body.configuration.fusebit_allowed_return_to}
 `,
-        nodejs: {
-            files: {
-                'package.json': {
-                    engines: {
-                        node: "10"
-                    },
-                    dependencies: {
-                        "@fusebit/add-on-sdk": "^0.0.1",
-                        "superagent": "^5.2.2",
-                        "googleapis": "39"
-                    }
-                },
-                'index.js': getTemplateFile('index.js'),
-                'initial.html': getTemplateFile('initial.html'),
-                'configure.js': getTemplateFile('configure.js'),
-            }
+    nodejs: {
+      files: {
+        'package.json': {
+          engines: {
+            node: '10',
+          },
+          dependencies: {
+            '@fusebit/add-on-sdk': '^0.0.1',
```

Comment:
Newer SDK?

Reply:
Won't fix, not touching the other addons, just prettifying.